pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -255,6 +264,35 @@ func UpdateEmbeddedEndpoint(podLister rv1.PodLister, serviceLister rv1.ServiceLi
 	return nil
 }
 
+func updateEmbeddedQuota(dep rv1beta2.DeploymentInterface, obj *v3.ClusterLogging) error {
+	d, err := dep.Get(loggingconfig.EmbeddedESName, metav1.GetOptions{})
```

Comment:
Use cache here?

Reply:
This function the only for update the deployment, so I get the real obj before the update.